config/http_config_test.go

```diff
@@ -317,7 +357,7 @@ func TestNewClientFromConfig(t *testing.T) {
 		}
 		response, err := client.Get(testServer.URL)
 		if err != nil {
-			t.Errorf("Can't connect to the test server using this config: %+v", validConfig.clientConfig)
+			t.Errorf("Can't connect to the test server using this config: %+v %v", validConfig.clientConfig, err)
```

Comment:
It's kind of hard to see that there is an error following after the config. How about:

"Can't connect to the test server using config %+v: %v"